spec/src/main/asciidoc/protocol-wireformat.adoc

```diff
@@ -212,10 +212,16 @@ Aspects regarding the secure access of health check information.
 | Readiness
 | See Appendix B
 
+| /health/start
```

Comment:
In the protocol-wireformat doc, we should also update the `Health Check Procedures` section to mention the new `Startup` health check procedure.